src/lib-components/forms/Checkbox.vue

```diff
@@ -2,16 +2,23 @@
   <div class="relative flex items-start">
     <div class="h-5 flex items-center">
       <input
-        type="checkbox"
-        class="focus:ring-blue-500 h-4 w-4 text-xy-blue border-gray-300 rounded disabled:opacity-50 disabled:cursor-not-allowed"
+        :aria-labelledby="label ? `${uuid}-label` : undefined"
         :checked="modelValue"
-        :disabled="disabled"
-        @change="$emit('update:modelValue', $event.target.checked)"
-        :required="required"
+        class="focus:ring-blue-500 h-4 w-4 text-xy-blue border-gray-300 rounded disabled:opacity-50 disabled:cursor-not-allowed"
+        :id="uuid"
+        type="checkbox"
+        v-bind="{
+          ...$attrs,
```

Comment:
🤑

Reply:
definitely comes with it's own set of concerns, but has a lot of potential